src/App.js

```diff
@@ -9,7 +10,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
+    <Article title={article.title} date={article.date} text={article.title} />
```

Comment:
https://kevinyckim33.medium.com/jsx-spread-operator-component-props-meaning-3c9bcadd2493

```suggestion
    <Article {...article} />
```